### PR TITLE
Combine aris slides with current curriculum

### PR DESCRIPTION
**Pull request by aneyzberg**

### CODE REVIEW

File: 02_Intro_to_Elixir/05-operators-and-variables.markdown [hunk 1]
```diff
@@ -3,8 +3,73 @@ layout: page
...
+c = div(b, a)
+```
+
+### How Assignment Actually works in Elixir
+```elixir
+iex> x = 2
+iex> y = 3
+```
+
+If we look at the code above. It looks like we have variables on the left-hand side of our expression and values on the right. And it appears that the values on the right are being "assigned" to the values on the left. However in elixir there is no assignment operator. The "=" is called the match operator. What does this mean?
```
Comment (aneyzberg) [inline 02_Intro_to_Elixir/05-operators-and-variables.markdown:58]: either expand on this or remove this section entirely.

File: 02_Intro_to_Elixir/05.7-maps-and-nested-data-structures.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: Maps and Nested Data Structures
+date: 2016-10-1 13:38:30 -0700
+---
+
+
+# Maps
+
+Maps in Elixir provides a quick key-value store called a map. Maps can be created by using the `%{}` , Keys in a map can be of any type.
```
Comment (aneyzberg) [inline 02_Intro_to_Elixir/05.7-maps-and-nested-data-structures.markdown:10]: reword this

File: 02_Intro_to_Elixir/05-operators-and-variables.markdown (hunk 1 above)
Comment (tdenkinger) [reply 02_Intro_to_Elixir/05-operators-and-variables.markdown:58]: I think we can address any concerns in conversation with the students. The "=" is a core and tricky concept, so we may want to dwell on it awhile during the training.

Comment (aneyzberg) [reply 02_Intro_to_Elixir/05-operators-and-variables.markdown:58]: yeah agreed. We actually go  into it in a later section.

Comment (tdenkinger) [reply 02_Intro_to_Elixir/05-operators-and-variables.markdown:58]: Right. I really like what's in this PR.

Comment (aneyzberg) [reply 02_Intro_to_Elixir/05-operators-and-variables.markdown:58]: I think this was left in from a prior version of the curriculum

**Pull request merged**